core/src/test/java/org/apache/calcite/test/enumerable/EnumerableCorrelateTest.java

```diff
@@ -143,13 +143,13 @@ class EnumerableCorrelateTest {
         .query(
             "select empid, name from emps e where exists (select 1 from depts d where d.deptno=e.deptno)")
         .explainContains(""
-            + "EnumerableCalc(expr#0..3=[{inputs}], empid=[$t0], name=[$t2])\n"
-            + "  EnumerableCorrelate(correlation=[$cor0], joinType=[inner], requiredColumns=[{1}])\n"
-            + "    EnumerableCalc(expr#0..4=[{inputs}], proj#0..2=[{exprs}])\n"
-            + "      EnumerableTableScan(table=[[s, emps]])\n"
+            + "EnumerableCalc(expr#0..5=[{inputs}], empid=[$t0], name=[$t2])\n  "
```

Comment:
Please move the spaces at the end of the previous line here.
Same in the subsequent lines.

Reply:
Is this case even changed?